### PR TITLE
Dev 2

### PR DESCRIPTION
**Pull request by akselatom**

### CODE REVIEW

File: DEV_2/DEV_2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                }
+
+                var inputString = args[0];
+                var replacer = new StringLettersToSoundReplacer();
+                inputString = replacer.VoicingOrStunningConsonantsReplacer(inputString);
+                inputString = replacer.ReplacementUnstressedO(inputString);
+                inputString = replacer.SofteningConsonants(inputString);
+                inputString = replacer.ProcessingVowelsIntoSounds(inputString);
+                Console.WriteLine(inputString);
+
```
Comment (AlexeiRomankov) [inline DEV_2/DEV_2/Program.cs]: please delete blank line

File: DEV_2/DEV_2/RussianPhoneticDictionary.cs [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    /// <summary>
+    /// The russian phoneme dictionary.
+    /// </summary>
+    public static class RussianDictionary
+    {
+        /// <summary>
+        /// A dictionary containing all possible soft consonants variants, except always soft consonants.
+        /// </summary>
+        public static readonly Dictionary<string, string> AllSoftSounds = new Dictionary<string, string>()
+                                                                       {
```
Comment (AlexeiRomankov) [inline DEV_2/DEV_2/RussianPhoneticDictionary.cs]: is it better to formate values on the left side?

**Review by AlexeiRomankov — COMMENTED** [review]
please do not use Russian language while commiting some code

File: DEV_2/DEV_2/Program.cs (hunk 1 above)
Comment (akselatom) [reply DEV_2/DEV_2/Program.cs]: Please confirm the changes for the possibility of merge to master

**Pull request merged**